locales-en-US.xml

```diff
@@ -249,6 +369,58 @@
     </term>
 
     <!-- LONG ROLE FORMS -->
+    <term name="chair">
+      <single>chair</single>
+      <multiple>chairs</multiple>
+    </term>
+    <term name="compiler">
+      <single>compiler</single>
+      <multiple>compilers</multiple>
+    </term>
+    <term name="contributor">
```

Comment:
@adam3smith What do you think? Should `contributor` just have the `with` verb form and no regular form, like with `author`?

Reply:
@bwiernik I'd leave it in for now. (I agree that this will be mostly be used like "Jane Smith, with John Doe. *A very important book* ...", but it won't hurt to have that in here as well. Ok?

Reply:
sure.